includes/blocks/course-theme/class-module-title.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * File containing the Module_Title class.
+ *
+ * @package sensei
+ * @since
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Display the title of the current module for the current lesson.
+ */
+class Module_Title {
+
+	/**
+	 * Module_Title constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/course-theme-module-title',
+			[
+				'render_callback' => [ $this, 'render' ],
+			]
+		);
+	}
+
+	/**
+	 * Renders the block.
+	 *
+	 * @param array $attributes The block attributes.
+	 *
+	 * @access private
+	 *
+	 * @return string The block HTML.
+	 */
+	public function render( array $attributes = [] ): string {
+		$lesson_id = \Sensei_Utils::get_current_lesson();
+		if ( ! $lesson_id ) {
+			return '';
+		}
+
+		$module_term = \Sensei()->modules->get_lesson_module( $lesson_id );
+		if ( ! $module_term ) {
+			return '';
+		}
+
+		$title = $module_term->name;
+		if ( ! $title ) {
+			return '';
+		}
+
+		return "<h1 class='sensei-course-theme-module-title'>{$title}</h1>";
```

Comment:
I think for the lesson title it makes sense to be `h1`, but in this case, we could use something less relevant. I think it could be an `em`, or something like that.

Reply:
I updated both `sensei-lms/course-theme-post-title` and `sensei-lms/course-theme-module-title` to support custom `tagName` and `className` attributes. Now we can render them in any tag that we need.

At the moment, the default tags for post title is `h1` and for module title is `h2`. Does that work?

Reply:
🤔 I think it would be interesting to use something else here. The headings are part of the same scope of headings, so it would mean that the module (`h2`) would be one level below (`h1`) lesson, which is not exactly true. I see the module name more as an "identifier" (or a type of content) than a title. Does it make sense?

Reply:
Yes. It does make sense. But I went through all HTML elements and tried to find a fitting element, but couldn't come up with an obvious answer. I think `p` could be the one. I'll change it to `p` for now. Once you let me know your preferred element for this I'll update it.

Reply:
I think `p` is not bad, but I'd go with another one that doesn't have a semantic representation since we don't have a tag that would represent that. Maybe a `div`, `span`? Or `em` / `strong` just to give a little relevance to that content?

Reply:
Changed it to `em`. 👍